### DISCUSSION TITLE
Running the example train_ppo.py -> KeyError

### DISCUSSION THREAD

**Discussion by Phorbol** [OpenRL-Lab/openrl#23] [Q&A] [answered]
![1683187232418](https://user-images.githubusercontent.com/75873382/236144977-07a48e5d-8dca-4c56-8042-c7c8ef1a88c8.png)
![1683187255231](https://user-images.githubusercontent.com/75873382/236144978-43ba9feb-6d65-4a0b-b20b-c07d5104a091.png)

My system is Ubuntu20.04, and python 3.8, 3.9 is all cannot work because of KeyError as above.

**Comment by WentseChen** [comment, marked as answer]
This bug has been fixed in #29 .
